src/unix/core.c

```diff
@@ -823,7 +823,7 @@ void uv__io_init(uv__io_t* w, uv__io_cb cb, int fd) {
 
 
 void uv__io_start(uv_loop_t* loop, uv__io_t* w, unsigned int events) {
-  assert(0 == (events & ~(POLLIN | POLLOUT | UV__POLLRDHUP)));
+  assert(0 == (events & ~(POLLIN | POLLOUT | UV__POLLRDHUP | POLLPRI)));
```

Comment:
Can you add a `UV__POLLPRI` define to src/unix/internal.h and set it to zero in case a platform does not have POLLPRI?